build/config/common-settings.json

```diff
@@ -119,6 +119,7 @@
    "lockscreen.locked": true,
    "lockscreen.unlock-sound.enabled": false,
    "message.sent-sound.enabled": true,
+   "moz.b2g.version": "3.0.0.0-prerelease",
```

Comment:
I don't fully understand why we even have 3.0.0.0-prerelease if the versions we use and display to the user are 2.1, 2.2, 2.3 (and 2.3m, 2.3s etc.)